llvm/docs/CodingStandards.rst

```diff
@@ -1713,6 +1713,14 @@ would help to avoid running into a "dangling else" situation.
     handleOtherDecl(D);
   }
 
+  // Use braces for the `else` block to keep it uniform with the `if` block.
+  if (isa<FunctionDecl>(D)) {
+    verifyFunctionDecl(D);
+    handleFunctionDecl(D);
+  } else {
```

Comment:
I would add an `else if` single-statement block too.

Reply:
Sounds good, I'll amend the example to include an else if and an else.